### PR TITLE
refactor: move everything out of the common folder

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR:
* deletes `src/common/constants.ts` (all in `src/models` now)
* deletes `src/common/txFlags.ts` (all in `src/models` now)
* moves `src/common/ecdsa.ts` up to the `src` level
* moves `src/common/errors.ts` up to the `src` level, and exports them at the top

### Context of Change

We don't really need a `common` folder if there's only going to be 2 files that should be in there - they can also fit at the `src` level. 

### Type of Change

- [x] Refactor (non-breaking change that only restructures code)

## Test Plan

CI passes.

### CODE REVIEW

**Review by JST5000 — COMMENTED** [review]
Should ecdsa.ts be moved into src/wallet since it's only used by functions which are (or will be moved into) in the Wallet folder?

**Comment by mvadari** [comment]
It's at the top level in xrpl-py, so I'm inclined to leave it where it is. If anyone else has opinions on this lmk

**Review by JST5000 — APPROVED** [review]
LGTM!

File: src/index.ts [hunk 1]
```diff
@@ -13,4 +13,6 @@ export * from './models/transactions'
 
 export * from './utils'
 
+export * from './errors'
```
Comment (natenichols) [inline src/index.ts:16]: Do we want to be exporting errors?

**Review by natenichols — APPROVED** [review]
I have one question. I don't really see the need to export `errors`, but also this is not a hill I'm willing to die on.

File: src/index.ts (hunk 1 above)
Comment (mvadari) [reply src/index.ts:16]: `catch` blocks or to check the error type

Comment (mvadari) [reply src/index.ts:16]: I'm happy to take it out, but we can discuss this more in my PR for exporting things at the top level (which I'll put up by tomorrow)

**Pull request merged**